optimizationProblem.m

```diff
@@ -15,23 +15,29 @@
     
     properties (Access = public)
         targetTensor = 1/3*eye(3); %Isotropic encoding tensor
-        N = 50;
+        N = 77;
```

Comment:
That is accurate. I removed old files because they could be confused. So to stay on the safe side my branch was purged.

To implement backward compatibility a button (or control) is needed to state if Maxwell effects are to be considered or not.